### ISSUE TITLE
IE8 problems

### ISSUE THREAD

**Issue by indvd00m** [rotaready/moment-range#240]
File [moment-range.js](https://cdnjs.cloudflare.com/ajax/libs/moment-range/3.1.1/moment-range.js) contains objects fields with name "default" and "for". This is broke working of moment-range in Internet Explorer 8 because of IE treat this fields as keywords.

**Comment by gf3** [comment]
hi! how are you including moment-range in your app? and are you using a build system?

**Comment by indvd00m** [comment]
No, there is no build system for client-side code in this project. File moment-range.js was downloaded from CDN and included as <script/> tag.

**Comment by TristanJM** [comment]
@indvd00m Are you including the files as follows?

```html
<script src="CDN for moment.js"></script>
<script src="CDN for moment-range.js"></script>

<script>
    window['moment-range'].extendMoment(moment);
    // ...
</script>
```

**Comment by indvd00m** [comment]
Yes, all included in accordence with documentation. Just open the file from first comment in this issue, line 137:
```javascript
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
```
This is same problem as here, for example: https://github.com/martinandert/counterpart/issues/6

As a temporary solution I escaped "default" and "for" keywords with quotes when them declared and with `['default']` when them used. Or it is possible just rename this fields.
